st_package_reviewer/check/file/check_command_names.py

```diff
@@ -0,0 +1,57 @@
+from . import FileChecker
+import re
+import ast
+
+class CheckCommandNames(FileChecker):
+    """Finds all sublime commands and does various checks on them."""
+
+    def check(self):
+        self.prefixes = set()
+        self.visit_all_pyfiles()
+        if len(self.prefixes) > 1:
+            self.warn("Found multiple command prefixes: {}. Consider using one single prefix so as"
+                " to not clutter the command namespace.".format(", ".join(self.prefixes)))
+
+    # TODO: This only checks immediate base classes; need more traversing for deeper-derived base
+    # classes.
+    def is_derived_from_command(self, node):
+        interesting = ("TextCommand", "WindowCommand", "ApplicationCommand", "ExecCommand")
+        for base in node.bases:
+            # print(ast.dump(base))
+            if isinstance(base, ast.Attribute):
+                # something of the form module_name.BaseClassName
+                if isinstance(base.value, ast.Attribute):
+                    if base.value.value.id == "Default":
+                        # Something derived from a class in Default... Must be ExecCommand
+                        return True
+                elif isinstance(base.value, ast.Name):
+                    if base.value.id == "sublime_plugin" and base.attr in interesting:
+                        return True
+            elif isinstance(base, ast.Name):
+                # something of the form BaseClassName
+                if base.id in interesting:
+                    return True
+        return False
+
+    def visit_ClassDef(self, node):
+        if not self.is_derived_from_command(node): return
+        # Check if the command is documented.
+        docstring = ast.get_docstring(node)
+        if not docstring:
+            with self.file_context(self.current_file):
+                self.warn("At line {}, column {}, the command {} does not have a docstring."
```

Comment:
I'm okay with both @FichteFoll and @keith-hall here. It'll have to change but I'm not sure if I would remove it completely or simply turn it into an option perhaps. I'll remove it for now because it's easy to bring it back in.